joomdonation/weblinks
# Real null values for datetimes and checked out user for weblinks 4.0

Pull Request for Issue # .

### Summary of Changes

Use real null values for datetimes/timestamps and for the checked_out column like we do in the CMS in version 4.

### Testing Instructions

Check if ...
- ... updating a present weblinks 3.x to 4.0 works on a Joomla 4 previously updated from 3.10 with MySQL /MariaDB and PostgreSQL databases.
- ... making a new installation of weblinks 4.0 on Joomla 4 works with MySQL /MariaDB and PostgreSQL databases.
- ... checked out icon works in the list view.
- ... checking in weblink items in the weblinks list view and with com_checkin works.
- ... published up and down works and the published icons in list views and possibly frontend editing are working.
- ... new items which never have been modified have modified time equal to created time and modified by user equal to created user in database.
- ... items which have been modified have the right modified time and user.
- ... anything else which comes into your mind works.

### Documentation Changes Required

None.
